File: src/my_first_node.cpp
#include <ros/ros.h>

int main(int argc, char** argv){
	ros::init(argc, argv, "my_fist_cpp_node"); // different node name
	ros::NodeHandle nh; // don`t exist in python

	ROS_INFO("Node has been started");

	ros::Rate rate(10); // in Hz

	while(ros::ok()){
		 // while the node is not shutdown
		 ROS_INFO("Hello");
		 rate.sleep();
	}

}

File: scripts/my_first_node.py
#!/usr/bin/env python3

import rospy

if __name__ == '__main__':
    rospy.init_node('my_fist_python_node')

    rospy.loginfo("This node has been started")

    rate = rospy.Rate(10) # rate at 10 Hz

    while not rospy.is_shutdown():
        rospy.loginfo("Hello")
        rate.sleep()
        
